pkg/codegen/schema/pulumi.json

```diff
@@ -110,6 +110,23 @@
                 "$ref": "#/$defs/token"
             }
         },
+	"dependencies": {
+	    "description": "A map from package name to version that describes the set of dependencies for this package.",
+	    "type": "array",
+	    "items": {
+		"type": "object",
```

Comment:
This should be a full package descriptor. So we'll need downloadURL and parameterisation parts as well.